static/app/views/performance/browser/webVitals/components/performanceBadge.tsx

```diff
@@ -0,0 +1,39 @@
+import {useTheme} from '@emotion/react';
+import styled from '@emotion/styled';
+
+type Props = {
+  status: 'good' | 'meh' | 'poor';
+};
+
+export const scoreToStatus = (score: number) => {
+  if (score >= 90) {
+    return 'good';
+  }
+  if (score >= 50) {
+    return 'meh';
+  }
+  return 'poor';
+};
+
+export function PerformanceBadge({status}: Props) {
+  const theme = useTheme();
+  const text = status === 'good' ? 'GOOD' : status === 'meh' ? 'MEH' : 'POOR';
+  const color =
+    status === 'good'
+      ? theme.green300
+      : status === 'meh'
+      ? theme.yellow300
+      : theme.red300;
```

Comment:
Another nit, feel free to ignore this but I personally find nested ternaries a real pain to read 🤣 

This would be a lot cleaner if we just mapped status values to a theme color instead via a constant, and pass in the status value to retrieve it

Reply:
same as above!